logstash-core/lib/logstash/agent.rb

```diff
@@ -86,7 +86,7 @@ def reload_state!
         begin
           reload_pipeline!(pipeline_id)
         rescue => e
-          @logger.error I18n.t("oops", :error => e, :backtrace => e.backtrace)
+          @logger.error I18n.t("oops"), :message => e.message, :class => e.class.name, :backtrace => e.backtrace
```

Comment:
ah, was confusing. Can we add braces around `@logger.error( I18n.t("oops"), :message => `